tests/functional/kmip/highlevel.spec.js

```diff
@@ -120,4 +122,130 @@ describe('KMIP High Level Driver', () => {
             done();
         });
     });
+
+    it('should retry without kmipHandshake on thales session revoked', async () => {
```

Comment:
Both tests use nearly identical setup (options, client init, healthcheck). could we factor out common setup into a shared helper or `beforeEach()` block to improve readability and reduce duplication. Example: `initTestClient()`.

Reply:
Fixed this file in [945dbac](https://github.com/scality/Arsenal/pull/2402/commits/945dbac1e9f510b39a7b9e61643a3bab2eae4cc8) and [b11e602](https://github.com/scality/Arsenal/pull/2402/commits/b11e6020627cf0321681b1d348a7194e0a3ec701)